aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/CallbackContext.java

```diff
@@ -39,4 +46,15 @@ public void setAddTagsComplete(final boolean addTagsComplete) {
     public ProbingContext getProbingContext() {
         return probingContext;
     }
+
+    public void timestampOnce(final String label, final Instant instant) {
```

Comment:
this looks like a copy of DBInstance's implementation. Does it make sense to move this to the common library?

Reply:
Done.